vault/mount.go

```diff
@@ -450,6 +450,21 @@ func (e *MountEntry) SyncCache() {
 	}
 }
 
+func (entry *MountEntry) Deserialize() map[string]interface{} {
```

Comment:
I think we typically prefer using single letters for receiver vars. Probably `e` here?

Reply:
Might also be worth tossing a comment in here

Reply:
This was from before, It just conflicted with my changes. I prefer not to change that at least in this PR.

Reply:
Ah, now that I am thinking why is it there. Oh man, brain damaged